### PR TITLE
Input messages

### PR DESCRIPTION
**Pull request by andrzejgorski**

### CODE REVIEW

File: whylog_vim/output_formater/teacher_formater/input_windows_messages.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    def get_constraint_message(cls, parsers):
+        output = OutputAggregator()
+        cls._add_message_prefix(ouptut, WindowTypes.INPUT)
+        for parser in parsers:
+            output.add_commented(ParserOutputs.LINE_CONTENT % (parser._id, parser.line_content))
+            output.add_commented(
+                ParserOutputs.META % (parser.line_resource_location, parser.line_offset)
+            )
+            output.add_commented(parser.pattern)
+            for group in parser.groups.keys():
```
Comment (kgromadzki) [inline whylog_vim/output_formater/teacher_formater/input_windows_messages.py]: Use six.iterkeys

File: whylog_vim/consts.py [hunk 2]
```diff
@@ -35,3 +35,6 @@ class EditorStates(object):
 class ActionTypes(object):
     STANDARD = 'standard action'
     TEACHER = 'teacher action'
+
+
+class ParserOutputs():
```
Comment (ppolewicz) [inline whylog_vim/consts.py]: inherit from object

File: whylog_vim/output_formater/consts.py [hunk 3]
```diff
@@ -3,3 +3,7 @@ class LogReader(object):
     LINE_HEADER = '--- %s [%s offset %s]:'
     EMPTY_OUTPUT = 'There is no cause of this line in config.'
     EMPTY_OUTPUT_CONTINUE = 'To add new rules press WhylogTeach (<F4> by default).'
+
+
+class Global(object):
+    EMPTY_LINE = ''
```
Comment (ppolewicz) [inline whylog_vim/output_formater/consts.py]: we don't really need a constant for this

File: whylog_vim/output_formater/teacher_formater/input_windows_messages.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        for group in six.iterkeys(parser.groups):
+            output.add_commented(
+                ParserOutputs.GROUP_CONVERTER %
+                (group, parser.groups[group].converter, parser.groups[group].content)
+            )
+        output.add_commented(ENDING)
+        return output
+
+    @classmethod
+    def get_constraint_message(cls, parsers):
```
Comment (ppolewicz) [inline whylog_vim/output_formater/teacher_formater/input_windows_messages.py:57]: this method and get_primary_key_message share a significant percentage of their code. Refactor.

**Pull request merged**